package.json

```diff
@@ -40,6 +40,7 @@
     "@google-cloud/common": "^0.11.0",
     "continuation-local-storage": "^3.1.4",
     "gcp-metadata": "^0.1.0",
+    "is": "^3.2.0",
```

Comment:
@AdriVanHoudt Indeed – it was my intention to not accept arrays as a valid 'bag of properties'. The `is.object` semantics were needed here.